src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-box-big {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-box-small {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
```

Comment:
Do not repeat your self. Check it all over the page

![2023-01-20_15h16_31](https://user-images.githubusercontent.com/61904995/213703664-64397944-0c0f-403a-a481-da8954e001d5.png)

Reply:
Done - ready for re-review.